game-using-jsDOM/styles.css

```diff
@@ -0,0 +1,268 @@
+    display:flex;
+    justify-content: center;
+    /* font-family: 'Elsie', cursive; */
+    font-family: cambria;
+    font-size: xx-large;
+}
+
+#word{
+    font-family: 'Great Vibes', cursive;
+    color: #8260a2;
+}
+
+.startButtonDiv {
+    display:flex;
+    justify-content: center;
+}
+
+.hideStartButton {
+    display: none;
+}
+
+.gameBoard {
+    display: flex;
+    justify-content: center;
+    /* background-color: red; */
+
+}
+.gameBoardDiv {
+    /* background-color: blue; */
+    width: 600px;
+    height: 400px;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-evenly;
+    align-items: center;
+}
+
+.letterBoxesDiv {
+    /* background-color: teal;  */
+    justify-content: center;
+    display: flex;
+    padding: 20px;
+}
+
+.letterBox {
+    /* border-color: orange; */
+    margin: 5px 5px 5px 5px;
+    box-shadow: rgba(60, 64, 67, 0.3) 0px 1px 2px 0px, rgba(60, 64, 67, 0.15) 0px 1px 3px 1px;
+    padding: 20px;
+    height: 15px;
+    width: 10px;
+    /* border-bottom: 1px solid black; */
+}
+
+#guessesSpan {
+    color: #36486b;;
+}
+
+.alphabetInnerDiv {
+    display: flex;
+    justify-content: center;
+}
+
+.alphabetButton {
+    height: 50px;
+    width: 50px;
+    border-radius: 10px;
+    margin: 2px 2px 2px 2px;
+    background-color: #dae5e9;
+    border: 1px solid #d5d9d9;
+    box-shadow: rgba(213, 217, 217, .5) 0 2px 5px 0;
+}
+
+.alphabetButton:hover {
+    background-color: #95b3be;
+}
+.alphabetButton:active {
+    background: #61a2ac;
+}
+
+.alphabetButton:disabled {
+    background-color: rgb(246, 243, 243);
+}
+
+.guessesDiv {
+    /* background-color: pink; */
+    box-shadow: rgba(213, 217, 217, .5) 0 2px 5px 0;
+    padding: 10px 10px 15px 15px;
+    border: 1px solid #d5d9d9;
+}
+
+.endScreenSection {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+}
+
+.endScreenTextDiv {
+    text-align: center;
+    padding-bottom: 50px;
+}
+
+#GameOver {
+    font-family: 'Notable', sans-serif;
+    font-size: 80px;
+    color: red;
+}
+
+#YouWon {
+    font-family: 'Notable', sans-serif;
+    font-size: 80px;
+    color: green;
+}
+/* Animation for Start Button
+ * Credit: https://codepen.io/chloe47632/pen/gOgxXGR
+ */
+
+.startButton, .startOverButton {
+	margin: auto;
+	border-radius: 20px;
+	background-color: #667292;
+	box-shadow: 10px 15px 40px #36486b;
+	width: 300px;
+	height: 100px;
+	font-size: 50px;
+	font-family: cambria;
+	color: #bccad6;
+	border: none;
+}
+
+.startButton:hover, .startOverButton:hover {
+	border: none;
+	background-color: #8d9db6;
+	opacity: 1;
+	color: #36486b;
+}
+
+.startButton, .startOverButton {
+	border: none;
+	background-color: #8d9db6;
+	opacity: 1;
+	color: #36486b;
+}
+
+#startButton:hover, #startOverButton:hover {
+	transition: 0.5s ease-in-out;
+	background: #8d9db6;
+}
+
+.startButton, .startOverButton {
+	--color1: #36486b;
+	--color2: #bccad6;
+	font-family: cambria;
+    /* font-family: 'Poppins', sans-serif; */
```

Comment:
remove comment

Reply:
see my note above!

Reply:
Thank you for your comments! I totally forgot to remove them and have now removed all the ones I could find.